### DISCUSSION TITLE
Pengujian Hipotesis - Membandingkan data tidak seimbang, dengan rentang waktu 1 tahun dan rentang waktu 2 tahun?

### DISCUSSION THREAD

**Discussion by Hertzspr** [teamalgoritma/community#63] [Q&A] [answered]
Misalkan ada data: lagu rilisan dari 2020, dan lagu dari 2021, dengan cutoff data akhir 2021. Jadi dua data tersebut memiliki rentang waktu berbeda, 1 tahun dan 2 tahun. Untuk melakukan statistical test perbandingan yang adil bagaimana? 

Kalau saya buat:
Hipotesis null = tidak ada perbedaan pada stream antara lagu rilisan tahun 2020 dan 2021, 
dan 
Hipotesis alternatif = ada perbedaan pada stream antara lagu rilisan tahun 2020 dan 2021, 

Apakah bisa dilaksanakan pengujian hipotesis dengan data tidak seimbang seperti itu? 
Sebelumnya saya mencoba mencari metode yang cocok untuk perbandingan 2 kelompok nonparametrik, sepertinya bisa dengan Mann Whitney U Test, apakah benar? Atau ada rekomendasi statistic test lain yang lebih tepat?

**Comment by inytss** [comment, marked as answer]
Dear @Hertzspr, 
Untuk pengujian statistik pada dua sampel yang memiliki jumlah data yang berbeda bisa menggunakan beberapa metode:
- Mann Whitney U test / Wilcoxon test
- T independent sample test
- Chi-squared test
- dan masih banyak lagi

Berikut artikel yang saya coba temukan untuk menjadi bahan bacaan:

- mengenai pilihan statistical test yang bisa digunakan pada data yang dimiliki https://towardsdatascience.com/statistical-testing-understanding-how-to-select-the-best-test-for-your-data-52141c305168
- macam-macam uji statistik untuk two sample https://en.wikipedia.org/wiki/Two-sample_hypothesis_testing
- test statistik untuk dua sample yang sering digunakan pada a/b testing https://towardsdatascience.com/a-b-testing-a-complete-guide-to-statistical-testing-e3f1db140499

Demikian jawaban saya, semoga membantu :)
